sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -150,12 +159,78 @@ _rom_ext_start_boot:
   la   sp, (_stack_end - 16)
 
   /**
-   * Set well-defined interrupt/exception handlers
-   *
-   * The lowest two bits should be `0b01` to ensure we use vectored interrupts.
+   * Initialize the Ibex simple address translation to run position dependent
+   * code on the virtual flash space.
    */
-  la   t0, (_rom_ext_interrupt_vector + 1)
-  csrw mtvec, t0
+  /**
+   * Compute the current slot address by: 
+   * addr = pc - _addr_translation_crt_offset
+   */
+  la t0, _addr_translation_crt_offset 
+  sub a0, a0, t0 //first address physical
+  // mask = virtual_addr | ((size - 1) >> 1);
+  la a1, _vflash_address
+  la t0, _vflash_size
+  addi t0, t0, -1
+  srli t0, t0, 1
+  or a1, a1,t0 
+
+  // Load ibex config base address
```

Comment:
nit: end all sentences with a period (`.`). It's a small thing but its part of our style guide.